DependencyInjection/Compiler/WellKnownSchemaFilterPass.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\DBAL\Configuration;
+use Symfony\Component\Cache\Adapter\PdoAdapter;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
+use Symfony\Component\Lock\Store\PdoStore;
+use Symfony\Component\Messenger\Transport\Doctrine\Connection;
+
+/**
+ * Blacklist tables used by well-known Symfony classes.
+ */
+class WellKnownSchemaFilterPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (! method_exists(Configuration::class, 'setSchemaAssetsFilter')) {
+            // only supported when using doctrine/dbal 2.9 or higher
+            return;
+        }
+
+        $blacklist = [];
+
+        foreach ($container->getDefinitions() as $id => $definition) {
+            if ($definition->isAbstract() || $definition->isSynthetic()) {
+                continue;
+            }
+
+            switch ($definition->getClass()) {
+                case PdoAdapter::class:
+                    $blacklist[] = $definition->getArguments()[3]['db_name'] ?? 'cache_items';
```

Comment:
Shouldn't it be `db_table`, here and below?

https://github.com/symfony/symfony/blob/3f759d7cf4c00df6694878c1a7db0df1551bd21b/src/Symfony/Component/Cache/Adapter/PdoAdapter.php#L35

https://github.com/symfony/symfony/blob/3f759d7cf4c00df6694878c1a7db0df1551bd21b/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php#L159

https://github.com/symfony/symfony/blob/3f759d7cf4c00df6694878c1a7db0df1551bd21b/src/Symfony/Component/Lock/Store/PdoStore.php#L60

Reply:
indeed, updated